backend/app/assets/javascripts/backend/controllers/backOfficeHome_controller.js

```diff
@@ -40,12 +48,31 @@
           this.indexPaginationView.hideLoader();
 
           if(response.status === 204 || response.status === 500) {
-            this.indexPaginationView.toggleBlockPagination();
+            this.indexPaginationView.blockPagination();
           }
         }.bind(this)
       });
     },
 
+    _indexSearcher: function(params) {
+      params = _.extend({}, params, { search: this.indexSearcherView.search });
+      var path = window.location.pathname.split("/");
+
+      $.ajax({
```

Comment:
I was wondering if we should try to use the Turbolinks.visit instead of simple AJAX, this way it would be reflected on the URL straight-away. Not sure how more complicated it would be at this stage. But might be worth checking on we ended up having the filters working on the front end.